.github/workflows/ci-cd.yml

```diff
@@ -55,8 +55,12 @@ jobs:
           set -e
           echo 'ENVIRONMENT=${{ vars.ENVIRONMENT }}' >> .env
           echo 'DOMAIN=${{ vars.DOMAIN }}' >> .env
-          echo 'SECRET_KEY=${{ secrets.SECRET_KEY }}' >> .env
-          echo 'CSRF_SECRET=${{ secrets.CSRF_SECRET_KEY }}' >> .env
+          echo 'REFRESH_TOKEN_EXPIRE_MINUTES=${{ vars.REFRESH_TOKEN_EXPIRE_MINUTES }}' >> .env
+          echo 'VERIFY_TOKEN_SECRET_KEY=${{ secret.VERIFY_TOKEN_SECRET_KEY }}' >> .env
+          echo 'ACCESS_TOKEN_SECRET_KEY=${{ secret.ACCESS_TOKEN_SECRET_KEY }}' >> .env
+          echo 'REFRESH_TOKEN_SECRET_KEY=${{ secret.REFRESH_TOKEN_SECRET_KEY }}' >> .env
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** CI/CD pipeline script includes hard-coded secret keys.

Ensure that secrets are securely managed and not hard-coded in CI/CD scripts. Use GitHub Secrets or similar secure storage.